core_contracts/dex/dex.py

```diff
@@ -441,6 +452,21 @@ def getDexOn(self) -> bool:
         """
         return self._dex_on.get()
 
+    @external(readonly=True)
+    def getTimeWeighting(self) -> bool:
+        """
+        Returns `True` if the dex has time weighted averages set for snapshots
+        """
+        return self._time_weighted_lp.get()
+
+    @only_governance
+    @external
+    def setTimeWeighting(self, _value: bool) -> bool:
+        """
+        Returns `True` if the dex has time weighted averages set for snapshots
```

Comment:
```py
def setTimeWeighting(self, _value: bool) -> bool:
```

That method doesn't return anything. It should be:
```py
def setTimeWeighting(self, _value: bool) -> None:
```

Also wrong method description in the docstring.